policy/release/slsa_build_scripted_build.rego

```diff
@@ -120,3 +152,15 @@ subject_digest(subject) := digest if {
 	some algorithm, value in subject.digest
 	digest := concat(":", [algorithm, value])
 }
+
+trusted_build_task_error(build_task_bundles) := error if {
+	count(build_task_bundles) == 0
+	error := "No Pipeline Tasks built the image"
+} else := error if {
+	untrusted := [task |
+		some task in bundles.unacceptable_task_bundle(lib.tasks_from_pipelinerun)
+		task.ref.bundle in build_task_bundles
+	]
+	count(untrusted) > 0
+	error := "Build Task is not trusted"
```

Comment:
Should the error say which task(s) were this?
```suggestion
	error := sprintf("Build Task(s) %v are not trusted", [untrusted])
```

Reply:
Yes, much better!